cattle/plugins/docker/compute.py

```diff
@@ -330,15 +330,17 @@ def get_container(self, client, instance, by_agent=False):
             return container
 
         if hasattr(instance, 'externalId') and instance.externalId:
-            f = lambda x: self._id_filter(instance.externalId, x)
```

Comment:
Had to change this because I got the following flake8 error in drone (though I get no flake8 errors locally):
`do not assign a lambda expression, use a def`